SSA/Projects/InstCombine/AliveHandwrittenLargeExamples.lean

```diff
@@ -6,6 +7,16 @@ open MLIR AST
 
 namespace AliveHandwritten
 
+namespace LLVMTheory
+
+@[simp]
+theorem LLVM.const?_eq : LLVM.const? i = .some (BitVec.ofInt w i) := rfl
+
+@[simp]
+theorem LLVM.xor?_eq : LLVM.xor? a b  = .some (BitVec.xor a b) := rfl
+
+end LLVMTheory
+
```

Comment:
Let's move these to where `LLVM.const?` and friends are defined, at https://github.com/opencompl/ssa/blob/efac4d7ad48716e345b2e52d48fc8b345333f4d7/SSA/Projects/InstCombine/LLVM/Semantics.lean#L411

Reply:
Done.